sdk/textanalytics/azure-ai-textanalytics/swagger/README.md

```diff
@@ -57,7 +57,7 @@ output-folder: $(python-sdks-folder)/textanalytics/azure-ai-textanalytics/azure/
 These settings apply only when `--tag=release_3_1` is specified on the command line.
 
 ```yaml $(tag) == 'release_3_1'
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.1/TextAnalytics.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/9820e5b189076cb9b329b6968a8d2fc405eef2f6/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.1/TextAnalytics.json
```

Comment:
Where did you get this commit? The last to touch that file I found was https://raw.githubusercontent.com/Azure/azure-rest-api-specs/0de25e2a040e1257b3a90faea868ad93c3435e48/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.1/TextAnalytics.json

Reply:
I took the last commit here: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.1/TextAnalytics.json